app/assets/javascripts/tasks_ratings.coffee

```diff
@@ -0,0 +1,74 @@
+ready =->
+  initializeRatings()
+
+$(document).on('turbolinks:load', ready)
+
+initializeRatings = ->
+  USERRATING = 0
+
+#  $('.popup-rating').hover (->
```

Comment:
this was in the old code - do you think it serves any purpose?
it seems to work fine without